### DISCUSSION TITLE
Recent Changes of UPnP (#3244?)

### DISCUSSION THREAD

**Discussion by udippel** [gerbera/gerbera#3250] [General]
No idea if this has to make with #3244?:
Very recently, in the Web GUI, some strange entries show up. At least, for me for the first time, the gerbera box itself. And, also for the first time, the fibre modem (gateway), despite of its UPnP being deactivated. 

 
![image](https://github.com/user-attachments/assets/47518f0d-bdd5-4f26-b316-3cc38a638daa)

**Comment by KarlStraussberger** [comment]
The server is normal to show up. You're gateway may still send out ssdp request although not responding to su requests itself.
